scripts/sct_register_to_template.py

```diff
@@ -282,6 +282,7 @@ def main(args=None):
     sct.run('sct_convert -i ' + fname_landmarks + ' -o ' + path_tmp + ftmp_label)
     sct.run('sct_convert -i ' + fname_template + ' -o ' + path_tmp + ftmp_template)
     sct.run('sct_convert -i ' + fname_template_seg + ' -o ' + path_tmp + ftmp_template_seg)
+    sct_convert.main(args=['-i', fname_template_vertebral_labeling, '-o', path_tmp + ftmp_template_label])
```

Comment:
Has this to do with `sct.display_viewer()` ?

Reply:
no. i fixed a bug that was on the way.